client/src/commands/code_analysis.ts

```diff
@@ -202,13 +202,22 @@ export const runCodeAnalysisWithReanalyze = (
   p.on("close", () => {
     diagnosticsResultCodeActions.clear();
 
+    let json: DiagnosticsResultFormat | null = null;
+
     try {
-      var json = JSON.parse(data);
+      json = JSON.parse(data);
     } catch (e) {
       window.showErrorMessage(
         `Something went wrong parsing the json output of reanalyze: '${e}'`
```

Comment:
Can you add a bit more. How about encouraging reporting the issue.

Reply:
I'll add this in a separate PR when I've figured out what makes sense to tell the user here.